build.gradle.kts

```diff
@@ -249,6 +249,7 @@ task("goIntegrationTests") {
 }
 
 task("playgroundPreCommit") {
+  dependsOn(":playground:lintProto")
```

Comment:
Is this triggered in both backend and frontend CIs?

Reply:
we have common proto files for our project located in playground/api/v1